task/sql-tasks.js

```diff
@@ -105,7 +154,22 @@ async function task_1_6(db) {
  *
  */
 async function task_1_7(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT  
+            e.EmployeeID as EmployeeId, 
+            concat(e.FirstName, " ", e.LastName)  as FullName,
+            CASE
+                WHEN e.ReportsTo is null
+                THEN "-"
+                ELSE concat(r.FirstName, " ", r.LastName)
+                END
+            as ReportsTo     
+        FROM northwind.Employees as e

```

Comment:
do not include schema name in the queries, it should be part of connection settings, Fix in all queries